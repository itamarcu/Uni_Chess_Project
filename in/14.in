game_mode 2
start
move <2,A> to <4,A>
move <7,G> to <6,G>
move <2,B> to <3,B>
save in/14.xml
quit
