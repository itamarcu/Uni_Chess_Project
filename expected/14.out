 Chess
-------
Specify game settings or type 'start' to begin a game with the current settings:
Game mode is set to 2-player
Starting game...
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| m _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| _ m m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B N R |
7| M M M M M M _ M |
6| _ _ _ _ _ _ M _ |
5| _ _ _ _ _ _ _ _ |
4| m _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| _ m m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M M M M M _ M |
6| _ _ _ _ _ _ M _ |
5| _ _ _ _ _ _ _ _ |
4| m _ _ _ _ _ _ _ |
3| _ m _ _ _ _ _ _ |
2| _ _ m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
Game saved to: in/14.xml
Enter your move (black player):
Exiting...
